src/benchmarks/micro/coreclr/PacketTracer/ObjectPool.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Collections.Concurrent
```

Comment:
I suggest removing this file as well and use array in the top-level function instead.

Reply:
good point, #done